src/by/it_academy/homeworks/hw4/DynamicArray.java

```diff
@@ -0,0 +1,157 @@
+package by.it_academy.homeworks.hw4;
+
+import java.util.Arrays;
+
+public class DynamicArray {
+    private int[] array;
+
+    public DynamicArray() {
```

Comment:
Лучше тут тоже массив проинициализировать - если использовать этот конструктор для теста

```
        DynamicArray test = new DynamicArray();
        test.print();
```

Получаю ошибку

```
Exception in thread "main" java.lang.NullPointerException: Cannot read the array length because "<local1>" is null
	at DynamicArray.print(DynamicArray.java:26)
	at Test.main(DynamicArray.java:130)
```